sealir-tutorials/Makefile

```diff
@@ -35,10 +35,11 @@ $(PAGES_SUBDIR):
 
 # Pattern rule to convert .ipynb files to rendered .html
 $(PAGES_SUBDIR)/%.html: %.ipynb $(PAGES_SUBDIR)
-	jupyter nbconvert --to html --output=$@ $<
+	jupyter nbconvert --execute --to html --output=$@ $<
 
-# Clean target to remove generated notebooks
+# Clean target to remove generated notebooks and rendered HTML
 clean:
-	rm -rf *.ipynb
+	rm -f $(IPYNB_FILES)
+	rm -f $(HTML_FILES)
```

Comment:
should this also remove `$(PAGES_SUBDIR)`?

Reply:
Good idea.  Fixed in 40461e8.  (I also learned that `rm -d` is like `rmdir`, but lets you add the `-f` option, so removing the directory when it doesn't exist doesn't raise an error in make.  🌈